Adds calls to new daru-io methods (read, to_format_string)

athityakumar opened the pull request:
Without these changes, Heroku build is currently failing as the methods (from, read, to, to_s, write) have been made distinctly in daru-io now.

athityakumar committed:
Adds calls to new daru-io methods (read, to_format_string)

zverok commented on app/controllers/sciruby_github_controller.rb:
```diff
@@ -6,12 +6,12 @@ class ScirubyGithubController < ApplicationController
-    respond_to do |format|
-      format.html  # just run the sciruby_github/repos.erb
-    end
+    @df_repos = Daru::DataFrame.read_json(
```
Let's remove the `df_` prefix? Ugly...

zverok commented on app/controllers/sciruby_github_controller.rb:
```diff
@@ -6,12 +6,12 @@ class ScirubyGithubController < ApplicationController
+      'https://api.github.com/orgs/Sciruby/repos',
+      Repo_name: '$..name',
+      forks: '$..forks',
+      open_issues: '$..open_issues_count'
```
How about keys consistency? E.g. `repo_name` (snake_case for all, or CamelCase for all)

zverok commented on app/helpers/sciruby_github_helper.rb:
```diff
@@ -2,25 +2,10 @@ module ScirubyGithubHelper
-      # end
+      format.csv  { send_data(dataframe.to_csv_string)                         }
+      format.xls  { send_data(dataframe.to_excel_string)                       }
+      format.json { send_data(dataframe.to_json_string(pretty: true))          }
```
:heart_eyes:

athityakumar committed:
Removes df_ prefix from dataframe variables, and adopts uniform snake_case

athityakumar committed:
Adds minor changes to HTML Importer dependencies

The light-weight Nokogiri gem has been used to support the HTML Importer, rather than the Mechanize gem. :tada:
